assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/mini-cart-footer-block/block.tsx

```diff
@@ -58,6 +61,9 @@ const Block = ( {
 		  parseInt( cartTotals.total_items_tax, 10 )
 		: parseInt( cartTotals.total_items, 10 );
 
+	// The buttons `Cart` and `Checkout` buttons were converted to inner blocks, but we still need to render the buttons
```

Comment:
The word `buttons` is duplicated:
```suggestion
	// The `Cart` and `Checkout` buttons were converted to inner blocks, but we still need to render the buttons
```